docs/en/week08/08-3.md

```diff
@@ -0,0 +1,266 @@
+---
+lang-ref: ch.08-3
+title: Generative Models - Autoencoders
+lecturer: Alfredo Canziani
+authors: Nandhitha Raghuram, Xinyi Zhao
+date: 25 March 2021
+---
+
+
+## Recap: Predictive Latent Variable Model
+
+As we know latent variables are useful in providing additional information that help in predicting multiple values of the target $\boldsymbol{y}$, assiociated with the input $x$. We have seen earlier, in order to predict the target variable $\boldsymbol{y}$ from latent variable $z$, the energy function is given as,
+
+$$
+E(\boldsymbol{y},z) = [y_1 - g_1(z)]^2 + [y_2 - g_2(z)]^2, \boldsymbol{y} \in \boldsymbol{Y}
+$$
+
+To be clear, we show the unit of $\boldsymbol{g}$:
+
+$$
+\boldsymbol{g} = [g_{1}, g_{2}]^\top, \mathbb{R} \rightarrow \mathbb{R}^{2} \\
+\boldsymbol{g}(z) = [w_{1}\cos(z), w_{2}\sin(z)]
+$$
+
+In this case, the the latent variable $z$ is one dimensional. However, if the $z$ has a higher dimension, this could lead to an energy function of $0$  everywhere. Hence, in order to ensure lower energy values for more compatible predictions we introduce a regularized factor. 
+
+
+### Training recap
+
+Given an observation $\boldsymbol{y}$, training the regualarized latent variable model
+
+$$
+E(\boldsymbol{y},z) = C(\boldsymbol{y},\boldsymbol{\tilde{y}}) + R(z)
+$$ 
+
+where $\boldsymbol{\tilde{y}} = \text{Dec}(z)$,
+
+
+- compute 
+
+$$
+F_{\beta}(\boldsymbol{y}) = \underset{z}{\text{softmin}_{\beta}}[E(\boldsymbol{y},z)]
+$$
+
+- Minimise 
+
+$$
+\mathcal{L}(F_{\beta}(\cdot), Y)
+$$
+
+At zero temperature limit, 
+- Compute
+
+$$
+\check{z} = \underset{z}{\arg\min}{E(\boldsymbol{y},z)} \\
+F_{\infty}(\boldsymbol{y}) = \underset{z}{\min}{E(\boldsymbol{y},\check{z})}
+$$
+
+- Minimise 
+
+$$
+\mathcal{L}(F_{\infty}(\cdot), Y)
+$$
+
+
+### Target propagation
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/target_prop.png" style="background-color:#DCDCDC;" /><br>
+<b>Figure 1:</b> Target propagation
+</center>
+
+Given an observation $\boldsymbol{y}$, we first compute $\tilde{z}$ which is the initial guess of the latent variable $z$
+
+$$
+\tilde{z} = \text{Enc}(\boldsymbol{y})
+$$
+
+We then initialize $z$ to $\tilde{z}$, here we must ensure that these values are not too far apart from each other. 
+
+$$
+\check{z} = \underset{z}{\arg\min}{E(\boldsymbol{y},z)} \\
+E(\boldsymbol{y},z) = C(\boldsymbol{y}, \boldsymbol{\tilde{y}}) + R(z) + D(z,\tilde{z})
+$$
+
+We now minimise the loss functional in two steps, minimise $\mathcal{L}(F_{\infty}(\cdot), Y)$:
+
+$$
+\theta_{\text{Dec}} = \theta_{\text{Dec}} - \triangledown_{\theta_{\text{Dec}}}C(\boldsymbol{y},\boldsymbol{\tilde{y}})\\ \\
+\theta_{\text{Enc}} = \theta_{\text{Enc}} - \triangledown_{\theta_{\text{Enc}}}D(\check{z},\tilde{z})
+$$
+
+In this way $z$ is constrained in taking only a set of values.
+
+
+## Autoencoder (AE)
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-3/AE.png" style="background-color:#DCDCDC;" /><br>
+<b>Fig. 2</b>: Autoencoder
```

Comment:
Same here. There is no caption, just title.

Reply:
added